sdk/resourcemanager/README.md

```diff
@@ -96,6 +96,36 @@ See [Authentication][authenticate] for more options.
 
 See [Samples][sample] for code snippets and samples.
 
+### Standalone packages
+
+The standalone packages, of which services are unavailable inside `azure-resourcemanager`, follow the same naming patterns and design principals.
+
+For example, to use the standalone package for media services, please include as below:
+
+<!-- TODO: package update start -->
+```xml
+<dependency>
+  <groupId>com.azure.resourcemanager</groupId>
+  <artifactId>azure-resourcemanager-mediaservices</artifactId>
+  <version>1.0.0-beta.1</version>
+</dependency>
+```
+<!-- TODO: package package end -->
+
+After above configuration, the manager can be authenticated by following code:
+
+<!-- TODO: embedme -->
+```java
+AzureProfile profile = new AzureProfile(AzureEnvironment.AZURE);
+TokenCredential credential = new DefaultAzureCredentialBuilder()
+    .authorityHost(profile.getEnvironment().getActiveDirectoryEndpoint())
+    .build();
+MediaservicesManager mediaservicesManager = MediaservicesManager.configure()
+    .authenticate(credential, profile);
+```
+
+Comparing to `azure-resourcemanager`, the standalone packages will have different release cycles, which mainly depend on the demand of Azure services. To check the full list of standalone packages with the consumed API versions, please see [Packages](docs/Packages.md).
```

Comment:
package list currently is in `### Service features`. Not sure is it need a `Packages.md`.

Reply:
The file name can be changed. But I will have this file as the output of the script to centralize the versions and package tags only for lite SDKs.

Reply:
Got it. Then it may need to deal with the `### Service features` words. (Separate this two list)

Reply:
Not sure whether it is worthwhile to demonstrate shared HttpClient with `AzureResourceManager`. Maybe later.

```
        // share the same http client and credential.
        HttpClient httpClient = HttpClient.createDefault();
        TokenCredential tokenCredential = new DefaultAzureCredentialBuilder().build();
        AzureProfile azureProfile = new AzureProfile(AzureEnvironment.AZURE);

        // init Fluent Premium
        AzureResourceManager azureResourceManager = AzureResourceManager.configure()
                .withHttpClient(httpClient)
                .authenticate(tokenCredential, azureProfile)
                .withDefaultSubscription();

        // init Fluent Lite media service
        MediaservicesManager mediaservicesManager = MediaservicesManager.configure()
                .withHttpClient(httpClient)
                .authenticate(tokenCredential, azureProfile);
```